InvenTree/InvenTree/apps.py

```diff
@@ -117,6 +117,12 @@ def start_background_tasks(self):
             schedule_type=Schedule.DAILY
         )
 
+        # Make regular backups
+        InvenTree.tasks.schedule_task(
```

Comment:
Can we give the user (or more likely, sysadmin) a method to customize:

a) Whether backups are performed automatically or not
b) The schedule period for performing backups

Reply:
a) Yes, easy
b) Not currently implemented in the registering logic so I'd rather not put this in here and wait if there are user requests for it